### DISCUSSION TITLE
"Cant handle compressed files with symlink downloader for download"

### DISCUSSION THREAD

**Discussion by jmrushing** [rogerfar/rdt-client#502] [Q&A] [answered]
**What version are you using?**
2.0.78

**Wat OS are you running?**
Gentoo (shared host)

**Are you using Docker or as a service?**
Service (via screen; systemd isn't available)

**Which debrid provider are you using?**
R-D

**Which downloader are you using?**
Symlink

**Please attach a log file here with the log setting set to debug**
2024-05-19 01:16:27.385 +00:00 [ERR] RdtClient.Service.Services.TorrentRunner: Download reported an error: Cant handle compressed files with symlink downloader for download 

I'm trying to use RDTClient's Symlink Downloader in conjunction with radarr/sonarr for requests and zurg + rclone to handle mounting R-D.
Why does it try to download a rar'ed file from R-D if the Symlink downloader can't handle it? Is this an issue with RDTClient or some misconfiguration with Zurg. I'm very confused...

**Comment by BrownQ8** [comment]
same here .. 
wnload reported an error: Cant handle compressed files with symlink downloader for download Fallout.S01.COMPLETE.2160p.AMZN.WEB-DL.DV.HDR10.PLUS.DDP5.1.Atmos.H265.MKV-BEN.THE.MEN.rar. Completed: 0%, avg speed: 0bytes/

**Comment by jamcalli** [comment]
This also started happening for me. Please advise if you find a solution.

**Comment by rogerfar** [comment, marked as answer]
The solution would probably be to not download all files from a torrent, then Real-Debrid won't try to zip it up but serve you a single file instead. Try setting the minimum file size in RDT.

**Reply by jmrushing** [reply]
I think your solution worked for that issue; the torrent is now changing to Finished. 
However, now I'm running into a problem in Radarr where the file is never imported due to an error in Radarr: "unable to determine if file is sample" along with some ffprobe error in the Event log.
I'm guessing this is due to it being a symlink? I can't find a Radarr settings that would be causing this...

**Reply by jmrushing** [reply]
Quick update to my last post: after more digging, the error from ffprobe was an "i/o error". Running ffprobe on the symlink file returns the same error; as does running ffprobe on the file inside the real debrid rclone mount. Now I'm thinking this is a permissions issue or the rclone mount is misconfigured. Any suggestions on rclone mount options I should be using?

**Reply by jmrushing** [reply]
Going to go ahead and mark as answer. My problem lies elsewhere (zurg and/or rclone mount). Thanks!

**Reply by EsmailELBoBDev2** [reply]
> The solution would probably be to not download all files from a torrent, then Real-Debrid won't try to zip it up but serve you a single file instead. Try setting the minimum file size in RDT.

it does not work with me, I indeed did not download all files from RD (I download music, albums) and yet it zipped it for me so this solution did not help. 
![image](https://github.com/rogerfar/rdt-client/assets/28893833/2858ce48-c55c-4905-937c-559ad1e094ca)
